Feature static asset deliver /2 (#34055)
dotnet/AspNetCore.Docs: aspnetcore/release-notes/aspnetcore-9/includes/blazor.md

<!-- UPDATE 10.0 Confirm or update -->

> Support for multiple Blazor Web Apps per server project will be considered for .NET 10 (November  2025).

Reviewer: Typically, I'd expect the comma. Let's consider rolling it into the sentence. It reads a bit better ...

```suggestion
Support for multiple Blazor Web Apps per server project is under consideration for .NET 10 in late 2025.
```

Reviewer 2: I would just change "(November 2025)" to "(late 2025)"
